### DISCUSSION TITLE
How does one backfill a huge selection of partitions (not all, but not few as well, tens of them)?

### DISCUSSION THREAD

**Discussion by ghost** [dagster-io/dagster#17518] [Q&A] [answered]
Is there a way to input a comma separated list to Dagit?

<img width="500" alt="image" src="https://github.com/dagster-io/dagster/assets/56590927/5c54a9ab-1214-49dd-82f5-4aab43c27c10">

**Comment by clairelin135** [comment, marked as answer]
Hi @ssfabiopicchi. For time-partitioned assets you can select ranges in the UI (via click and select). Unfortunately for other partitions defs types, you'll have to manually click.

It is possible to submit graphQL requests where you can input a list of partitions, though these graphQL APIs are used internally and thus subject to changes without public notice.
